tests/unit/conftest.py

```diff
@@ -102,3 +102,14 @@ class DummyTaskOfflineBatchGeneration(DummyTask):
 @pytest.fixture
 def dummy_llm() -> AsyncLLM:
     return DummyAsyncLLM()
+
+
+@pytest.fixture
+def local_llamacpp_model_path():
+    """
+    Fixture that provides the local model path for LlamaCpp testing.
+
+    Returns:
+        str: The path to the local LlamaCpp model file.
+    """
+    return "./tests/model/gguf/all-MiniLM-L6-v2-Q2_K.gguf"
```

Comment:
do you clean this up after the tests?

Reply:
Thank you @davidberenstein1957  for your comments. I have updated the test. 
It will look for a model in location defined by environment variable `LLAMACPP_TEST_MODEL_PATH`. If the env variable is not set, the test will download a small model from hub , execute the test and then perform cleanup. It uses the `tmp_path` fixture provided by pytest to get a temporary directory for storing the downloaded model.